### PR TITLE
#100 Removed tabs on questions page

### PR DESCRIPTION
**Pull request by Belize22**
- Allows for easier merging.

I did my best to account for styling when fixing merge conflicts but verify thoroughly just incase.

### CODE REVIEW

File: SA3/qa_web/templates/qa_web/question_thread.html [hunk 1]
```diff
@@ -24,52 +24,51 @@ <h1 class="masthead-heading mb-0">
                 </span>
             </h1>
             <h5 class="masthead-subheading mb-0">
-                <p><span class="marked" id="content_question">{{ currentQuestion.content }}</span></p>
+                <p><span class="marked" id="content_question">
+                    {{ currentQuestion.content }}
+                </span></p>
```
Comment (alvyn279) [inline SA3/qa_web/templates/qa_web/question_thread.html]: Here, the span tag should be all on one line as it was before. The spaces and line breaks within the `span` tag makes our code vulnerable to the bug we had before: Scrollbars appearing on the content if MD processed it as `<code>`.

File: SA3/qa_web/templates/qa_web/question_thread.html [hunk 2]
```diff
@@ -24,52 +24,51 @@ <h1 class="masthead-heading mb-0">
...
             {% for comment in q_comments %}
                 <p>
-                    <span class="marked">{{ comment.content }}</span>
-                    (Comment by: {{ comment.owner }}. Date posted: {{ comment.creation_date }})
-                     <a id='upvote_{{ comment.id }}_comment'
-                        class="{% if comment.id in pos_v_c %}voted{% endif %}">
-                         <button>Upvote</button>
-                     </a>
+                    <span class="marked">
+                    {{ comment.content }}</span> (Comment by:
```
Comment (alvyn279) [inline SA3/qa_web/templates/qa_web/question_thread.html]: Same issue with this span.

Comment (alvyn279) [reply SA3/qa_web/templates/qa_web/question_thread.html]: Please fix all the occurences, or rebase accordingly to what we have on master.

File: SA3/qa_web/views.py [hunk 3]
```diff
@@ -280,15 +280,14 @@ def answers(request, id_):
...
     return render(request, 'qa_web/question_thread.html',
-                  {'currentQuestion': q, 'answers': q_answers, 'bestAnswer': q_best_answer, 'q_comments': q_comments,
-                   'a_comments': a_comments, 'initial_select_value': initial_select_value,
-                   'pos_v_q': pos_vote_q, 'neg_v_q': neg_vote_q,
-                   'pos_v_a': pos_vote_a, 'neg_v_a': neg_vote_a,
-                   'pos_v_c': pos_vote_c, 'neg_v_c': neg_vote_c})
+                  {'currentQuestion': q, 'answers': q_answers,
+                   'bestAnswer': q_best_answer, 'q_comments': q_comments,
+                   'a_comments': a_comments,
+                   'initial_select_value': initial_select_value})
```
Comment (alvyn279) [inline SA3/qa_web/views.py]: Not the complete context like we have on `master`...

**Comment by Lercerss** [comment]
Coverage is lower due to removing lines in `views.py`, therefore it can be safely ignored.

**Pull request merged**